docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -931,3 +932,62 @@ function su_humsci_profile_update_9709() {
   \Drupal::service('module_installer')->install(['paragraphs_features']);
   \Drupal::service('module_installer')->install(['paragraphs_ee']);
 }
+
+/**
+ * Uninstall and remove config for hs_row paragraph type and modules.
+ */
+function su_humsci_profile_update_9710() {
+  // Remove existing paragraph entities.
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $paragraph_machine_name = 'hs_row';
+
+  $query = $paragraph_storage->getQuery()
+    ->condition('type', $paragraph_machine_name)
+    ->accessCheck(FALSE);
+  $paragraph_ids = $query->execute();
+
+  if ($paragraph_ids) {
+    $paragraphs = Paragraph::loadMultiple($paragraph_ids);
+    foreach ($paragraphs as $paragraph) {
+      $paragraph->delete();
+    }
+
+    $message = t('Deleted all paragraphs of type @type.', [
+      '@type' => $paragraph_machine_name,
+    ]);
+    Drush::logger()->notice($message);
+  }
+
+  // Delete the field and field storage on the paragraph.
+  $fields = \Drupal::entityTypeManager()
+    ->getStorage('field_config')
+    ->loadByProperties(['bundle' => $paragraph_machine_name]);
+
+  if ($fields) {
+    foreach ($fields as $field) {
+      /** @var Drupal\field\Entity $field */
+
+      // Delete the field configuration (instance) from the paragraph type.
+      $field->delete();
+
+      // Load the corresponding field storage.
+      $field_storage = FieldStorageConfig::load($field->getName());
+      if ($field_storage) {
+        // Check if the field storage is used elsewhere.
+        if (!$field_storage->isLocked()) {
```

Comment:
I'm not sure if that's what `isLocked()` does in this scenario. The [$locked](https://api.drupal.org/api/drupal/core%21modules%21field%21src%21Entity%21FieldStorageConfig.php/property/FieldStorageConfig%3A%3Alocked/10) property only makes action unavailable in the UI and not the API.

It might be better to call out these field names explicitly rather than pulling them directly off the paragraph. The only field storage config I see deleted off the row is `field_hs_row_components`

Reply:
Fixed. We're removing the other `field_paragraph_style` instances in #1622. If that PR gets merged before this one, I'll update it to remove its storage here too, because that field is not used anywhere else.